packages/grid/src/mouse-handlers/GridLinkMouseHandler.ts

```diff
@@ -0,0 +1,256 @@
+/* eslint class-methods-use-this: "off" */
+
+import clamp from 'lodash.clamp';
+import { assertNotNull } from '@deephaven/utils';
+import { isEditableGridModel } from '../EditableGridModel';
+import { EventHandlerResult } from '../EventHandlerResult';
+import Grid from '../Grid';
+import GridMouseHandler, { GridMouseEvent } from '../GridMouseHandler';
+import GridRange from '../GridRange';
+import GridRenderer from '../GridRenderer';
+import GridUtils, { GridPoint } from '../GridUtils';
+
+export interface LinkInformation {
+  left: number;
+  top: number;
+  right: number;
+  bottom: number;
+  width: number;
+  height: number;
+  value: string;
+  href: string;
+}
+
+class GridLinkMouseHandler extends GridMouseHandler {
```

Comment:
I'd like to make this more generic, not `Link` specific. Could just name it `TokenMouseHandler` instead, then it'll just trigger when a token is clicked.